### ISSUE TITLE
PConv with Faster RCNN

### ISSUE THREAD

**Issue by wangjue-wzq** [jshilong/SEPC#22]
I use PConv in Faster RCNN to train my custom dataset and the mAP is only about 20, but I can get 90 mAP using FPN only. In Table 4 of the paper, you have done a comparative experiment on faster RCNN. Is there any error in my code? Looking forward for your response, thank you! The configs as fllow:
`

neck=[dict(
        type='FPN',
        in_channels=[256, 512, 1024, 2048],
        out_channels=256,
        start_level=1,
        add_extra_convs=True,
        num_outs=5),
        dict(
            type='SEPC',
            out_channels=256,
            Pconv_num=4,
            pconv_deform=False,
            lcconv_deform=False,
            iBN=False,  # when open, please set imgs/gpu >= 4
        )]

`
I change the sepc code to cater Faster RCNN as fllow:

`
@NECKS.register_module
class SEPC(nn.Module):

    def __init__(self,
                 in_channels=[256] * 5,
                 out_channels=256,
                 num_outs=5,
                 pconv_deform=False,
                 lcconv_deform=False,
                 iBN=False,
                 Pconv_num=4,

                 ):
        super(SEPC, self).__init__()
        assert isinstance(in_channels, list)
        self.in_channels = in_channels
        self.out_channels = out_channels
        self.num_ins = len(in_channels)
        self.num_outs = num_outs
        assert num_outs == 5
        self.fp16_enabled = False
        self.iBN = iBN
        self.Pconvs = nn.ModuleList()

        for i in range(Pconv_num):
            self.Pconvs.append(PConvModule(in_channels[i], out_channels, iBN=self.iBN, part_deform=pconv_deform))

        self.lconv = sepc_conv(256, 256, kernel_size=3, dilation=1, part_deform=lcconv_deform)
        # self.cconv = sepc_conv(256, 256, kernel_size=3, dilation=1, part_deform=lcconv_deform)
        self.relu = nn.ReLU()
        if self.iBN:
            self.lbn = nn.BatchNorm2d(256)
            # self.cbn = nn.BatchNorm2d(256)
        self.init_weights()

    # default init_weights for conv(msra) and norm in ConvModule
    def init_weights(self):
        # for str in ["l", "c"]:
        for str in ["l"]:
            m = getattr(self, str + "conv")
            init.normal_(m.weight.data, 0, 0.01)
            if m.bias is not None:
                m.bias.data.zero_()

    @auto_fp16()
    def forward(self, inputs):
        assert len(inputs) == len(self.in_channels)
        x = inputs
        for pconv in self.Pconvs:
            x = pconv(x)
        # cls = [self.cconv(level, item) for level, item in enumerate(x)]
        loc = [self.lconv(level, item) for level, item in enumerate(x)]
        if self.iBN:
            # cls = iBN(cls, self.cbn)
            loc = iBN(loc, self.lbn)
        # outs = [[self.relu(s), self.relu(l)] for s, l in zip(cls, loc)]
        outs = [self.relu(s) for s in loc]
        return tuple(outs)
`

**Comment by jshilong** [comment]
Sorry, I just noticed this issue.
we usually set `start_level` as `0` in two stage FPN

**Comment by wangjue-wzq** [comment]
OK！ Thank you！I will try.
